compiler/src/dotty/tools/dotc/reporting/diagnostic/messages.scala

```diff
@@ -5,10 +5,9 @@ package diagnostic
 
 import dotc.core._
 import Contexts.Context, Decorators._, Symbols._, Names._, NameOps._, Types._
-import util.{SourceFile, NoSource}
-import util.{SourcePosition, NoSourcePosition}
+import util.SourcePosition
```

Comment:
No the one below shouldn't have changed, I've fixed that. That might have been the IDE trying to tidy up for me